// [[Reason: undefined]] ////

//// [tests/cases/conformance/expressions/unaryOperators/negateOperator/negateOperatorInvalidOperations.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,15 +1,16 @@
 
 
 //// [negateOperatorInvalidOperations.d.ts]
-declare var NUMBER1: any;
+declare var NUMBER1: invalid;
 declare var NUMBER: any;
 declare var NUMBER2: number;
 declare var NUMBER3: number;
 declare var NUMBER4: number;
-declare var NUMBER: any;
+declare var NUMBER: number;
 
 /// [Errors] ////
 
+negateOperatorInvalidOperations.ts(4,14): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 negateOperatorInvalidOperations.ts(4,15): error TS1109: Expression expected.
 negateOperatorInvalidOperations.ts(4,30): error TS1005: ',' expected.
 negateOperatorInvalidOperations.ts(4,31): error TS1109: Expression expected.
 negateOperatorInvalidOperations.ts(7,17): error TS18050: The value 'null' cannot be used here.
@@ -21,13 +22,15 @@
 negateOperatorInvalidOperations.ts(12,5): error TS2403: Subsequent variable declarations must have the same type.  Variable 'NUMBER' must be of type 'any', but here has type 'number'.
 negateOperatorInvalidOperations.ts(12,14): error TS1109: Expression expected.
 
 
-==== negateOperatorInvalidOperations.ts (11 errors) ====
+==== negateOperatorInvalidOperations.ts (12 errors) ====
     // Unary operator -
     
     // operand before -
     var NUMBER1 = var NUMBER: any-;  //expect error
+                 
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                   ~~~
 !!! error TS1109: Expression expected.
                                  ~
 !!! error TS1005: ',' expected.
